test/gtest/uct/ib/test_ud_slow_timer.cc

```diff
@@ -97,7 +97,7 @@ UCS_TEST_P(test_ud_slow_timer, ep_destroy, "UD_TIMEOUT=1s") {
     EXPECT_TRUE(ucs_ptr_array_lookup(&iface->eps, ep_idx, ud_ep_tmp));
 
     m_e1->destroy_eps();
-    twait(ucs_time_to_msec(iface->config.peer_timeout * 1.5));
+    twait(ucs_time_to_msec(iface->config.peer_timeout * 3));
```

Comment:
can we wait for a long time (e.g 60 sec) **or** ep is not present in the array any more?

Reply:
hmm... currently it's unconditional waiting, no reason to wait 60 sec always, will wrap up in a loop with 60 sec timeout.